php-checks/src/main/java/org/sonar/php/checks/formatting/IndentationCheck.java

```diff
@@ -77,15 +80,24 @@ public void visitFunctionCall(FunctionCallTree tree) {
 
     if (check.isFunctionCallsArgumentsIndentation && !check.isInternalFunction(tree.callee())) {
       SyntaxToken calleeLastToken = ((PHPTree) tree.callee()).getLastToken();
-
       checkArgumentsIndentation(
-        tree.arguments(),
+        callArgumentToExpression(tree.callArguments()),
```

Comment:
```suggestion
        tree.callArguments().stream().map(CallArgumentTree::value).collect(Collectors.toList()),
```
And make the first parameter of `checkArgumentsIndentation` of type `List<? extends Tree>`. It seems to me that there is no need to pass a `SeparatedList` there, since nothing is done with the separators.

Then you can remove `callArgumentToExpression`